doctr/transforms/modules/pytorch.py

```diff
@@ -53,3 +53,15 @@ def __repr__(self) -> str:
         if self.preserve_aspect_ratio:
             _repr += f", preserve_aspect_ratio={self.preserve_aspect_ratio}, symmetric_pad={self.symmetric_pad}"
         return f"{self.__class__.__name__}({_repr})"
+
+
+class RandomGaussianNoise():
+    def __init__(self, mean=0.5, std=1.5):
+        self.std = std
+        self.mean = mean
+
+    def __call__(self, tensor):
+        return tensor + torch.randn(tensor.size()) * self.std + self.mean
```

Comment:
This won't work: if the input tensor is an image, there are two cases:
- the image is in float, in range [0, 1]
- the image is in uint8 in range [0, 255]

in both cases, we want the resulting image to stay in that range.

Additionally, depends how you see the noise addition but:
- take a single pixel in RGB, we don't want a shift, we want variance around its initial value
so this would be (meaning that self.mean should be 0 by default)
```python
if x.dtype == torch.uint8:
    return (x + 255 * (self.mean + self.std * torch.rand(x.shape, device=x.device))).round().clamp(0, 255).to(dtype=torch.uint8)
else:
    return (x + self.mean + self.std * torch.rand_like(x)).clamp(0, 1)
```

Reply:
This is only because we perform operations on images and that we want to preserve their value range integrity